<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>1163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WynPlan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>255</width>
         <height>82</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/resources/images/logo.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>918</width>
         <height>1000</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QFrame" name="frame_7">
          <property name="minimumSize">
           <size>
            <width>900</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>50</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_warning">
             <property name="minimumSize">
              <size>
               <width>800</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(192, 28, 40);</string>
             </property>
             <property name="text">
              <string>no hay profesor por favor vaya a la ventana de profesor a añadir profesores epsum lambda ...</string>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QFrame" name="frame_6">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QLabel" name="config_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(70, 70, 122);</string>
             </property>
             <property name="text">
              <string>AÑADIR PLAN DE ESTUDIO:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QPushButton" name="config_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/resources/images/multiple.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QFrame" name="frame_4">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>180</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QLabel" name="studyplan_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(70, 70, 122);</string>
             </property>
             <property name="text">
              <string>PLAN DE ESTUDIO</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QPushButton" name="studyplan_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/resources/images/studyplan.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>180</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QLabel" name="teacher_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(70, 70, 122);</string>
             </property>
             <property name="text">
              <string>PROFESOR</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QPushButton" name="teacher_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/resources/images/teacher.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>180</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QLabel" name="schedule_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(70, 70, 122);</string>
             </property>
             <property name="text">
              <string>HORARIOS</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QPushButton" name="schedule_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/resources/images/schedule.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QFrame" name="frame_5">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>180</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QLabel" name="section_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(70, 70, 122);</string>
             </property>
             <property name="text">
              <string>SECCIÓN</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
            <widget class="QPushButton" name="section_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/resources/images/section.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
